reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -229,6 +229,8 @@ abstract class TableReporter : Reporter() {
                         append(resolutions.joinToString(prefix = "\nResolved by: ") { "${it.reason} - ${it.comment}" })
                     }
                 }
+
+        fun isResolved() = !resolutions.isEmpty()
```

Comment:
Use `isNotEmpty()`. Also make it `val resolved` instead, as `resolutions` is immutable.